packages/app/components/Queue/StudentQueueList.tsx

```diff
@@ -212,8 +213,28 @@ export default function StudentQueueList({
       setHasDraftInProgress(false);
       finishQuestion(text, qt, isOnline, location);
       closeEditModal();
+      if (isFirstQuestion) {
+        notification.warn({
```

Comment:
It would be nice if this notification had a mouse hand to make it clear that you can click on it. Like `cursor: pointer`